src/Elastic.Apm/Api/ISpan.cs

```diff
@@ -71,26 +38,5 @@ public interface ISpan
 		/// Examples: 'db', 'external'.
 		/// </summary>
 		string Type { get; set; }
```

Comment:
There is a `string Type` both here and on `ITransaction`, but those are very different (`Transaction.Type != Span.Type`), so I'd prefer to keep those separate from the common interface.